calblueprint/girls-write-now
# [saved] Saved stories page

[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

- Added the library page with a LibraryHeader component displaying username and settings button
- Added toggle bar between favorites list and reading list to display story preview cards of user's favorite and reading list

## Relevant Links

### Notion Sprint Task

[//]: # 'Add the relevant Notion link(s) here'

https://www.notion.so/calblueprint/saved-Saved-stories-page-33f5d4e671204e7eb635d445714c2e63

### Online sources

[//]: # 'Optional - copy links to any tutorial or documentation that was useful to you when working on this PR'

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## How to review

[//]: # 'The order in which to review files and what to expect when testing locally'

- Log in with an account that already has an existing favorite and reading list, or manually add stories to the account's lists
- Go to library page and check that stories are displayed correctly, and clicking on story cards navigates to that story
- Check that toggling between reading and favorites list does so

## Next steps

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

## Tests Performed, Edge Cases

[//]: # 'Hopefully we will add a testing suite/CI soon, but until then note down the steps you took to test locally'

### Screenshots

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

https://github.com/calblueprint/girls-write-now/assets/156966341/e9c7ac8a-0be7-451a-97fa-148c6be4d996

CC: @adityapawar1

[//]: # 'This tags in Adi as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
